pallets/capacity/src/lib.rs

```diff
@@ -126,6 +127,11 @@ pub mod pallet {
 		/// The number of Epochs before you can unlock tokens after unstaking.
 		#[pallet::constant]
 		type UnstakingThawPeriod: Get<u16>;
+
+		/// Maximum number of blocks an epoch can be
+		/// currently used as the actual value of epoch length.
+		#[pallet::constant]
+		type MaxEpochLength: Get<Self::BlockNumber>;
```

Comment:
don't know if we should keep this name or not, it's not _technically_ premature optimization...

Reply:
I would drop it, but we can update it in a different PR.